snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -126,12 +126,12 @@ public void ListObjects()
         }
 
         [Fact]
-        public void DownloadFile()
+        public void DownloadObject()
```

Comment:
naw. I don’t feel strong, either. I just wanted you to know that I struggled, too. : )